tests/plugin_http_api_test.py

```diff
@@ -60,6 +62,28 @@ def startEnv(self) :
         self.nodeos.launchCmd(start_nodeos_cmd, self.node_id)
         time.sleep(self.sleep_s)
 
+    def activateAllBuiltinProtocolFeatures(self):
+        self.nodeos.activatePreactivateFeature()
+
+        contract = "eosio.bios"
+        contractDir = "unittests/contracts/old_versions/v1.7.0-develop-preactivate_feature/%s" % (contract)
+        wasmFile = "%s.wasm" % (contract)
+        abiFile = "%s.abi" % (contract)
+
+        producerKeys = Cluster.parseClusterKeys(0)
```

Comment:
Why 0 for the argument `totalNodes`. It could be the reason of the test failure.

Reply:
I think the argument is the number of additional producers other than the main one, this test is only using one producer, so that was the reason of 0. Anyways, for some reason this test was passing locally (probably using a leftover directory config from a previous execution), but then i started seeing the problem locally too, so probably the method doesn't work well if the nodeos instance is not set through the Cluster class.

In the end, I realized that the only reason i used that method from the Cluster class was to get the public and private keys for the eosio account, but as these get a default value if not defined explicitly, i just opted to use those instead.